DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,7 @@ public function getConfigTreeBuilder()
             ->scalarNode('piwik_host')->isRequired()->end()
             ->scalarNode('tracker_path')->defaultValue('/js/')->end()
             ->scalarNode('site_id')->isRequired()->end()
+            ->scalarNode('disable_cookies')->defaultValue(null)->end()
```

Comment:
Do you need the explicit `defaultValue()` call?

Reply:
Yes, otherwise an error occures that the value is not set.